.github/workflows/node.yml

```diff
@@ -3,10 +3,10 @@ name: Node
 on:
   push:
     branches: [ master ]
-    paths-ignore: ['swift/**', '*.podspec', 'java/**']
+    paths-ignore: ['swift/**', '*.podspec', 'java/**', 'rust/bridge/**']
   pull_request:
     branches: [ master ]
-    paths-ignore: ['swift/**', '*.podspec', 'java/**']
+    paths-ignore: ['swift/**', '*.podspec', 'java/**', 'rust/bridge/**']
```

Comment:
rust/bridge/node/ is relevant, so we'll have to be more fine-grained. Or switch to an include/exclude model (https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#example-using-positive-and-negative-patterns-1).

I do wish we could share variables across workflows, oh well.

Reply:
Oh my mistake, I forgot there is a `rust/bridge/node` now. I should also update the Swift and Java builds to ignore that and modify this to ignore ffi/jni but not node.

Reply:
Fixed